.ci/Brewfile.travis

```diff
@@ -0,0 +1,6 @@
+brew "pcre"
+brew "ccache"
+brew "cmake"
+brew "swig"
+brew "zeromq"
+brew "boost@1.69"
```

Comment:
Are we still testing with 1.61 and 1.58 on some of the builds?

Reply:
GCC 4.9 is using 1.61; it looks like 1.58 was used on the old XCode. Should we switch Clang 3.6 to Boost 1.58?

Reply:
It might be good to have at least one of the builds (daily or otherwise) use it since we do claim to support it.

Reply:
I'm switching the Clang 3.6 build to use 1.58 -- I don't know how accurate it is, but I feel like usually if a system has Clang installed, it is more up to date than GCC.